### PR TITLE
Correction of all assignments

### PR DESCRIPTION
**Pull request by ML6503**

### CODE REVIEW

File: assignments/logging-error-handling/assignment.md [hunk 1]
```diff
@@ -6,6 +6,8 @@ Your task is to implement logging functionality to already existing REST service
 
 ## Technical requirements
 
+- Task can be implemented on Typescript
+- Use [template](https://github.com/rolling-scopes-school/nodejs-course-template/generate)
```
Comment (AlreadyBored) [inline assignments/logging-error-handling/assignment.md:10]: Same as above

File: assignments/template-rest/assimnent.md [hunk 2]
```diff
@@ -4,7 +4,7 @@
 
 Let's try to create a Home Library Service!
 
-NB! You must create new repository from template for this task. Its name must be nodejs2022Q1-service i.e. full link to the repository must be https://github.com/%your-gihub-id%/nodejs2022Q1-service.
+NB! You must create new repository from [template](https://github.com/rolling-scopes-school/nodejs-course-template/generate) for this task. Its name must be nodejs2022Q1-service i.e. full link to the repository must be https://github.com/%your-gihub-id%/nodejs2022Q1-service.
```
Comment (AlreadyBored) [inline assignments/template-rest/assimnent.md:7]: 2022Q2

File: assignments/template-rest/score.md [hunk 3]
```diff
@@ -3,16 +3,16 @@
...
+- **+15** The application code that worked with `albums` instance divided into modules according to to its purpose (e.g.: work with request and response in *.router.js, business logic *.service.js, work with data storage in *.repository.js etc.)
+- **+15** The application code that worked with `movies` instance divided into modules according to to its purpose (e.g.: work with request and response in *.router.js, business logic *.service.js, work with data storage in *.repository.js etc.)
+- **+15** The application code that worked with `books` instance divided into modules according to to its purpose (e.g.: work with request and response in *.router.js, business logic *.service.js, work with data storage in *.repository.js etc.)
+- **+15** The application code that worked with `favourites` instance divided into modules according to to its purpose (e.g.: work with request and response in *.router.js, business logic *.service.js, work with data storage in *.repository.js etc.)
```
Comment (AlreadyBored) [inline assignments/template-rest/score.md:11]: We should follow Nest.js architecture conventions, so let's say something like `divided into modules by its purpose according to Nest.js architercture rules` (just example)

File: assignments/template-rest/score.md [hunk 4]
```diff
@@ -3,16 +3,16 @@
...
 
 ## Advanced Scope
 - **+30** Task implemented on Typescript
```
Comment (AlreadyBored) [inline assignments/template-rest/score.md:14]: Nest.js uses TS out of the box, please, remove this line

**Review by AlreadyBored — CHANGES_REQUESTED** [review]

File: assignments/authentication/assignment.md [hunk 5]
```diff
@@ -11,30 +11,28 @@ Your task is to implement Authentication and Authorization with JWT (Access and
...
+- Use [template](https://github.com/rolling-scopes-school/nodejs-course-template/generate)
 - Use 16 LTS version of Node.js
 
-
 ## Implementation details
 
-# REST service: Authentication & Authorization
 
-1. Implement **POST** `api/auth/signup` method (instead of **POST** `api/users`) that accepts **JSON** `username`, `password`, `age`, `hobbies` and server responses with HTTP **200** code and relevant message on success.
+1. Implement **POST** `api/auth/signup` method (instead of **POST** `api/users`) that accepts **JSON** `login`, `password` and server responses with HTTP **200** code and relevant message on success.
```
[the remark this answers is not in the document]
Comment (ML6503) [reply assignments/authentication/assignment.md:21]: As decided waiting for final sample Library API project to make all amendments accordingly

File: assignments/template-rest/score.md (hunk 4 above)
Comment (ML6503) [reply assignments/template-rest/score.md:14]: removed

File: assignments/template-rest/score.md (hunk 3 above)
Comment (ML6503) [reply assignments/template-rest/score.md:11]: Amended by your sample. Thank you

File: assignments/template-rest/assimnent.md (hunk 2 above)
Comment (ML6503) [reply assignments/template-rest/assimnent.md:7]: corrected

File: assignments/logging-error-handling/assignment.md (hunk 1 above)
Comment (ML6503) [reply assignments/logging-error-handling/assignment.md:10]: removed

File: assignments/authentication/score.md [hunk 6]
```diff
@@ -1,13 +1,13 @@
 # Scoring: Authentication & Authorization
 
-## NB! For the succesfull tests results user with username = **admin**, and password = **admin** should be saved into DB. 
+## NB! For the succesfull tests results user with login = **admin**, and password = **admin** should be saved into DB.
```
[the remark this answers is not in the document]
Comment (ML6503) [reply assignments/authentication/score.md:3]: removed

File: assignments/authentication/assignment.md (hunk 5 above)
[the remark this answers is not in the document]
Comment (ML6503) [reply assignments/authentication/assignment.md:21]: removed

File: assignments/authentication/assignment.md [hunk 7]
```diff
@@ -11,30 +11,28 @@ Your task is to implement Authentication and Authorization with JWT (Access and
 
 ## Technical requirements
 
-- Task can be implemented in Javascript or Typescript
-- Only `bcrypt` or its equivalent like `bcryptjs`, `jsonwebtoken`, `ws`, `robotjs`, `nodemon`, `dotenv`, `cross-env`, `typescript`, `ts-node`, `eslint` and its plugins, `webpack` and its plugins, `prettier`, `uuid`, `@types/*` as well as libraries used for testing are allowed
+- Task can be implemented in Typescript
+- Use [template](https://github.com/rolling-scopes-school/nodejs-course-template/generate)
```
[the remark this answers is not in the document]
Comment (ML6503) [reply assignments/authentication/assignment.md:15]: removed

File: assignments/authentication/assignment.md [hunk 8]
```diff
@@ -11,30 +11,28 @@ Your task is to implement Authentication and Authorization with JWT (Access and
 
 ## Technical requirements
 
-- Task can be implemented in Javascript or Typescript
-- Only `bcrypt` or its equivalent like `bcryptjs`, `jsonwebtoken`, `ws`, `robotjs`, `nodemon`, `dotenv`, `cross-env`, `typescript`, `ts-node`, `eslint` and its plugins, `webpack` and its plugins, `prettier`, `uuid`, `@types/*` as well as libraries used for testing are allowed
+- Task can be implemented in Typescript
```
[the remark this answers is not in the document]
Comment (ML6503) [reply assignments/authentication/assignment.md:14]: removed

**Pull request closed without merging**